src/content/drafts/eslint-community-org.md

```diff
@@ -0,0 +1,131 @@
+  [@not-an-aardvark](https://github.com/not-an-aardvark) transferred this repo
+  to the new org, but they're still maintaining it
+* [`eslint-plugin-promise`](https://github.com/eslint-community/eslint-plugin-promise)  
+  We took this one over from [@xjamundx](https://github.com/xjamundx)
+* [`eslint-plugin-security`](https://github.com/eslint-community/eslint-plugin-security)  
+  We took this one over from the
+  [@nodesecurity](https://github.com/nodesecurity) team, but
+  [@nzakas](https://github.com/nzakas) is still maintaining it
+* [`eslint-utils`](https://github.com/eslint-community/eslint-utils) and
+  [`regexpp`](https://github.com/eslint-community/regexpp)  
+  We created a fork from [@mysticatea](https://github.com/mysticatea)'s repos and
+  published them under the
+  [`@eslint-community` npm organization](https://npmjs.com/org/eslint-community).
+
+We also intend to fork, bring up to date, and publish the following packages
+originally authored by [@mysticatea](https://github.com/mysticatea) as well:
+
+* [`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es)  
+  We'll work on getting all
+  [`eslint-plugin-es-x`](https://github.com/eslint-community/eslint-plugin-es-x)
+  features merged as well
+* [`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments)
+* [`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node)  
+  We'll work on getting all
+  [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n)
+  features merged as well
+
+Of course, we don't want to stop there.
+
+If you think your project would be a good fit for the new community org, *or*
+you're depending on a project that's currently unmaintained and want to have the
+community to take it over, you can contact the Community Core team on
+[the official ESLint Discord](https://eslint.org/chat).
+
+Keep in mind that we'll only be able to take over projects that are adhering to
+the following acceptance criteria:
+
+* Is it a package that is ESLint-related?  
+  Mostly this will be ESLint plugins, but (unmaintained) dependencies of such
+  packages, closely related packages or packages split from the main ESLint repo
+  (like
+  [`eslint-formatter-codeframe`](https://github.com/eslint-community/eslint-formatter-codeframe)
+  or
+  [`eslint-formatter-table`](https://github.com/eslint-community/eslint-formatter-table))
+  or used by the main repo (like
+  [`eslint-utils`](https://github.com/mysticatea/eslint-utils) and
+  [`regexpp`](https://github.com/mysticatea/regexpp)) could go in the
+  `@eslint-community` GitHub organization as well.
+
+  Because plugins can expose their own (recommended) configs, we already accept
+  config packages in some way.  
+  However, the community core team won't be eager to accept shared ESLint config
+  packages, as they can be very opinionated.
+
+  It also doesn't matter if the package is purely TypeScript-related or not, as
```

Comment:
I'm not sure I understand this sentence. Will we accept purely TypeScript-related packages or not? What type of fragmentation are we worried about?

Reply:
We **do** accept purely TS-related packages

If we wouldn't, that could mean a `typescript-eslint-community` organization could be created by the community, which will only fragmentize the community & that's not something we want

Reply:
Okay, then I think we can just remove this sentence. I think the natural inclination will be to believe that any ESLint-related package can be a candidate, so I don't think we need to get this specific.

Reply:
Should all be fine now 👍